projects/ng-core-tester/src/app/app-routing.module.ts

```diff
@@ -208,7 +208,13 @@ const routes: Routes = [
           canUpdate,
           canDelete,
           canRead,
-          aggregations
+          aggregations,
+          searchHeaders: {
+            'Content-Type': 'application/rero+json'
+          },
+          detailHeaders: {
```

Comment:
Sounds like a bad name. You can use similar name than the invenio-records-rest API convention, i.e. list, item.

Reply:
done